doc/api/url.md

```diff
@@ -237,6 +237,22 @@ url.resolve('http://example.com/', '/one')    // 'http://example.com/one'
 url.resolve('http://example.com/one', '/two') // 'http://example.com/two'
 ```
 
+`url.resolve` behaves differently based on the protocol of the URL in the
+`from` parameter. Protocols like `http`, `https`, `file`, and `ftp` are treated
+specially to match browser behavior, like so.
```

Comment:
If possible, it would be helpful if the docs explained why it is like this. Is it ever useful, or is it flat-out browser bug that node emulates? Also, "Protocols like `http`, `https`, `file`, and `ftp`" is not helpful. How does a protocol become "like" those? I couldn't even hazard a guess. What is the actual list? Is there one?